cosmo_tester/framework/util.py

```diff
@@ -219,12 +221,13 @@ def get_auth_header(username=None, password=None, token=None):
 def create_rest_client(manager_ip,
                        manager_username=None,
                        manager_password=None):
+    headers = get_auth_header(
```

Comment:
Tenant is mandatory for every api. But the auth mechanism is based on user & password, not tenant.. It's like a required parameter..  
Community edition is using the default_tenant.